resources/views/site/operations/sectors.blade.php

```diff
@@ -316,12 +316,13 @@
                         sector you are in.
                     </p>
 
-                    <p>
-                        <strong>Primary Sectors</strong>
-                    </p>
+                    <h2>
+                        Information for Controllers
+                    </h2>
 
                     <p>
-                        For the purpose of controller bookings, the &#39;Primary Sectors&#39; are defined as:
+                        For the purpose of <a "{{ route('site.atc.bookings') }}" rel="">controller bookings</a>, 
```

Comment:
```suggestion
                        For the purpose of <a href="{{ route('site.atc.bookings') }}">controller bookings</a>, 
```

Reply:
Good with me! 

href error is my fault. 

rel="" was taken from elsewhere though! https://github.com/VATSIM-UK/core/blob/master/resources/views/site/atc/progression.blade.php#L22

Reply:
Yeah, it doesn't really make any difference, and in this context it doesn't do anything. Not sure why it is above!